.github/workflows/release.yml

```diff
@@ -100,23 +108,24 @@ jobs:
         id: changelog
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          current-tag: ${{env.PLUGIN_VERSION}}
+          current-tag: ${{ inputs.release_version }}
```

Comment:
shouldnt the tag be matching the version that is set to `gradle.properties`?

Reply:
yes, it is the same version, we set it at the very first steps.